go/private/library.bzl

```diff
@@ -218,7 +226,7 @@ def emit_go_compile_action(ctx, sources, libs, lib_paths, direct_paths, out_obje
     args += ["-src", src]
   for dep in direct_paths:
     args += ["-dep", dep]
-  args += ["--", "-o", out_object.path, "-trimpath", ".", "-I", "."]
+  args += ["-o", out_object.path, "-trimpath", ".", "-I", "."]
```

Comment:
I think the Travis failure is due to removing "--" here. That needs to come before `gc_goopts` to prevent those options from being parsed.